### DISCUSSION TITLE
Help needed: Build interactive REDCap structure for COVID F2/F3 v2 forms

### DISCUSSION THREAD

**Discussion by mllerch** [naccdata/uniform-data-set#43] [Help Needed]
There seems to be interest in using the pre-filled format for UDSv4. We could start by building the COVID forms as a pre-filled version drawing from the previous results to allow only new symptoms to be recorded. The form wasn't exactly designed to work this way, but I think with REDCap we could improve upon the original. Would anyone be interested in trying it? @Jreade89 or @dzemlock123 are either of you interested? 

Thanks for considering it!

**Comment by dzemlock123** [comment]
Hi Melissa, sorry for the delay. I agree that the UDSv4 forms should have information piped from one visit to the next when applicable. I can work on adding this feature to the COVID forms! Do we want all the variables in the COVID forms to be piped or just a few?

**Comment by mllerch** [comment]
Wonderful! I really appreciate your help working on this project. We would just want a subset of questions ported over like vaccinations and hospitalizations. I can mark-up specifically what we want. As your building it, you can provide feedback about the auto-populated content to us.
